src/middlewares/index.ts

```diff
@@ -1,3 +1,5 @@
 import * as Error from "./error";
+import { validate } from "./validation";
+import { SIGNUP_VALIDATORS, LOGIN_VALIDATORS } from "./validators";
```

Comment:
Caps are usually used for constants, change this if you still feel caps are required.

Reply:
Okay. Actually `*_VALIDATORS` is an array of middlewares. So to distinguish from other middleware, CAPS is used.
The array will be a constant though since middlewares wont change.